docs/cookbook.md

```diff
@@ -471,3 +471,9 @@ Extend Temporal to support arbitrarily-large years (e.g., **+635427810-02-02**)
 An example of using `Temporal.TimeZone` for other purposes than a standard time zne.
 
 → [NYSE time zone](cookbook-nyse.md)
+
+### Time zone directly from tzdata rules
+
+An example of building your own `Temporal.TimeZone` object from [tzdata-compatible rules](https://data.iana.org/time-zones/tz-how-to.html).
+
+→ [Time zone from tzdata](cookbook-tzdata.md)
```

Comment:
Any reason to this is split into a separate md file? The other examples all embed directly.

Reply:
Two reasons: one is that these last few examples are much larger than the other cookbook snippets, so for practicality they can live on their own pages. The other reason is that these last few examples are not so much intended as "copy this recipe if you want to do X" but more like "here's something esoteric that Temporal is capable of, that you probably won't need to do, but you might adapt this recipe for something else".